### PR TITLE
page updates for the NetBeans 24 release

### PR DESCRIPTION
**Pull request by mbien**

### CODE REVIEW

File: modules/ROOT/pages/download/index.adoc [hunk 1]
```diff
@@ -37,11 +37,11 @@ for important requirements for download pages for Apache projects.
 
 Apache NetBeans is released four times a year. For details, see link:https://cwiki.apache.org/confluence/display/NETBEANS/Release+Schedule[full release schedule].
 
-== Apache NetBeans 23
+== Apache NetBeans 24
 
-Latest version of the IDE, released on September 19, 2024.
+Latest version of the IDE, released on November 25, 2024.
```
Comment (mbien) [inline modules/ROOT/pages/download/index.adoc]: todo

File: modules/ROOT/pages/download/nb24/index.adoc [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+[start=2]
+. Follow the build instructions in the README
+
+== Community Approval
+
+As in any other Apache Project, the Apache NetBeans Community approved this release
+through the following voting processes in our xref:community/mailing-lists.adoc[mailing lists]:
+
+- link:https://lists.apache.org/thread/rvw8f43lgbpjmpqmdwq1cbkcnpf3ryyv[PMC vote]
+- link:https://lists.apache.org/thread/5lq00jf5r8gwl4mt6qmmy5xzn6okf7fg[PMC vote result]
```
Comment (mbien) [inline modules/ROOT/pages/download/nb24/index.adoc]: todo

File: modules/ROOT/pages/download/index.adoc (hunk 1 above)
Comment (ebarboni) [reply modules/ROOT/pages/download/index.adoc]: december 10, we keep the year that's cool

Comment (mbien) [reply modules/ROOT/pages/download/index.adoc]: will fix in a moment

Comment (mbien) [reply modules/ROOT/pages/download/index.adoc]: note for future me: there is a second date string on the DL page

**Review by ebarboni — APPROVED** [review]
LGTM

**Pull request merged**